homeassistant/const.py

```diff
@@ -779,9 +779,20 @@ class UnitOfMass(StrEnum):
 # Rotational speed units
 REVOLUTIONS_PER_MINUTE: Final = "rpm"
 
+
+# Irradiance units
+class UnitOfIrradiance(StrEnum):
+    """Irradiance units."""
+
+    WATTS_PER_SQUARE_METER = "W/m²"
+    BTUS_PER_HOUR_SQUARE_FOOT = "BTU/(h⋅ft²)"
```

Comment:
The unit changed compared to `IRRADIATION_BTUS_PER_HOUR_SQUARE_FOOT`, should this be handled?

Reply:
Thanks to EQUIVALENT_UNITS this should not be a breaking change.

Reply:
Right looked for it, and even then, I missed it; time for a break I guess 🙈